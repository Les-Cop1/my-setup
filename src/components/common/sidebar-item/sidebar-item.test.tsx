import React from 'react'

import { SidebarItem } from '@components'
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'

import { MemoryRouter } from 'react-router-dom'

describe('Sidebar item', () => {
  test('Snapshot', () => {
    const { container } = render(<SidebarItem label="Hello" to="/world" />, { wrapper: MemoryRouter })
    expect(container).toMatchSnapshot()
  })
})
