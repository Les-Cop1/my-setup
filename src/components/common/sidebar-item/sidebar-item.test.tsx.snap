// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar item Snapshot 1`] = `
<div>
  <a
    class="text-sm text-slate-300 hover:bg-slate-700 hover:text-white group flex items-center px-2 py-2 font-medium rounded-md"
    href="/world"
  >
    Hello
  </a>
</div>
`;
